qcodes/utils/installation_info.py

```diff
@@ -67,6 +67,11 @@ def convert_legacy_version_to_supported_version(ver: str) -> str:
     numbers to a regular version string. This is done by replacing a char
     by its ASCII code (using ``ord``). This assumes that the version number
     only uses at most a single char per level and only ASCII chars.
+
+    It also splits off anything that comes after the first ``_`` in the version str.
```

Comment:
dash, not underscore
```suggestion
    It also splits off anything that comes after the first ``-`` in the version str.
```